src/main/kotlin/org/web3j/evm/EmbeddedEthereum.kt

```diff
@@ -348,7 +359,7 @@ class EmbeddedEthereum(configuration: Configuration, private val operationTracer
         defaultBlockParameter: String
     ): String {
         // TODO error case..? See EthCall in Besu..
-        return makeEthCall(web3jTransaction, defaultBlockParameter)!!.output.hexString
+        return makeEthCall(web3jTransaction, defaultBlockParameter)!!.output.toHexString()
```

Comment:
I think the `!!` is unnecessary because `makeEthCall` is not returning a nullable type. Or if `makeEthCall` can be null, we should return `TransactionProcessingResult?`... Not sure which one is correct

Reply:
I modified this to return `TransactionProcessingResult?` if incase something goes wrong and returns null.